src/db.rs

```diff
@@ -198,6 +198,10 @@ impl Database {
                 debug!("output:\n{}", chunk);
             }
             if !column_names.is_empty() && !output.is_empty() {
+                if output.len() > 1 {
+                    let chunks: Vec<DataChunk> = output.drain(1..).collect();
+                    output[0].stack(&chunks);
```

Comment:
We can let `stack` accpets an iterator so the `collect` is not needed?

Reply:
here too, i try the code `output[0].stack(output.drain(1..))` and `output[0].stack(&output.iter())` first, and the complier. warns me that there are mutable borrows and one immutable borrows.
So i first stacks the vector into one temp variaible chunk, and then move the chunk to output[0]

Reply:
oh I understand your problem. I think there are several ways

First your current solution looks good to me. But I think we can use the temp variable directly without moveing it into output[0]?

Alternately we can also write things like 
```rust 
let it=output.into_iter();
let chunk=it.next().unwrap();
chunk.stack(it)
```

```